res/database/Default/database_people.xml

```diff
@@ -15,11 +15,10 @@
 			<nick_name></nick_name>
 			<details job="1" gender="1" birthday="" deathday="" country="" fictional="1" />
 		</person>
-		<person id="21e88c65-0cc6-4bd1-bd14-b48b6ce25402" tmdb_id="0" imdb_id="" creator="5578" created_by="Ronny">
+		<person id="21e88c65-0cc6-4bd1-bd14-b48b6ce25402" face_code="2:2:2:#EDA657:-1:-1:1:-1:7#C808A8:6:2:-1:3:3:7:13#581410:-1:-1" tmdb_id="0" imdb_id="" creator="5578" created_by="Ronny">
```

Comment:
So richtig wohl fuehl ich mich nicht dabei, `face_code` bei <person> einzuhaengen.

Wir haben ja `<details>` und `<data>` zur Verfuegung. `<details>` waere momentan ja eher "personality" (also Attribute etc) und `<details>` beschreibt eher so "Allgemeines".
Ich denke `face_code` koennte durchaus in den `<details>` - node wandern (es sollten ja sowohl Attribute als weider Kindernodes moeglich sein).

Reply:
Grund der Verschiebung direkt nach person ist, dass es aktuell auch für insignificantPerson (Ronny Otto, Själe) verwendet wird. Wenn man die ohne Kindknoten behalten möchte, geht es nicht anders (wenn man *eine* definierte Stelle für den face_code haben möchte).

Reply:
Hmmm stimmt ... vielleicht ein Grund, warum ich es an mehreren Stellen eingeplant hatte. "gender" etc wuerde ja ebenfalls dort mit hingehoeren .. und fictional und ...

Reply:
Ja gender, fictional, country, job sind auch nochmal in den details. Da finde ich die Dopplung aber vertretbar. Schließlich sind das Eigenschaften, die man bei beiden Personengruppen regelmäßig definieren will und bei Hauptpersonen ist die Lesbarkeit in den Details dann einfach höher.
face_code wird aber so selten verwendet, dass ich die Dopplung für überflüssig hielt (wie immer Einlesecode und Formatbeschreibung müssen da nicht 100% übereinstimmen).

Wäre die face_code-Behandlung so also in Ordnung?

Reply:
ich wuerde denken, wir sollten `face_code` wie "gender, fictional, country" ... ansehen - sprich auch an zwei Stellen erlauben ("Dopplung"). Nur ein `<image ....`-support waere wohl nicht wirklich noetig.

Reply:
OK. Dann passe ich den Datenbankaufräum-Commit entsprechend an und verschiebe den face_code der Celebrities nach details.

Reply:
hm, oder vielleicht lieber nach data? Schließlich sieht man im Spiel das Gesicht dort, wo man auch Humor, Aussehen etc. prüft.